.gitignore

```diff
@@ -113,3 +113,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+#for vscode
+.vscode
```

Comment:
Please add some extension to auto-fix pep8 issues or atleast highlight them in your editor itself.

Reply:
what would you suggest for vscode?

Reply:
Aah sorry! My comment was for the whole PR and not for this line.

Reply:
No Like I mean any extension for pep8 in vscode

Reply:
Yeah, you can look for `pycodestyle` in the VSCode marketplace. There are others as well, which you can search for.